src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -456,16 +469,17 @@ ApiGlobe.prototype.getHeading = function() {
  */
 
 ApiGlobe.prototype.getRange = function() {
+    return this.scene.currentControls().getRange();
+};
+
+ApiGlobe.prototype.getRangeFromEllispoid = function() {
 
+    // TODO: error is distance is big with ellipsoid.intersection(ray) because d < 0
```

Comment:
Typo in the method name `getRangeFromEllispoid` → `getRangeFromEllipsoid`
Also : no documentation? what is the difference between getRange and `getRangeFromEllipsoid`? why/when should I use one or the other?